testsuite/MDAnalysisTests/core/test_atomgroup.py

```diff
@@ -1008,7 +1008,7 @@ class TestUnwrapFlag(object):
         'moment_of_inertia': np.array([[16687.941, -1330.617, 2925.883],
                                        [-1330.617, 19256.178, 3354.832],
                                        [2925.883,  3354.832, 8989.946]]),
-        'asphericity': 0.2969491080,
+        'asphericity': np.array([0.61 , 0.701, 0.381]),
```

Comment:
As above, why is this now an array vs scalar?

Reply:
Same as above

Reply:
See response above :)